js/client-chat.js

```diff
@@ -1237,6 +1237,13 @@
 					this.$chat.append('<div class="notice">' + Tools.sanitizeHTML(row.slice(1).join('|')) + '</div>');
 					break;
 
+				case 'notify':
+					if (!Tools.prefs('mute') && Tools.prefs('notifvolume')) {
+						soundManager.getSoundById('notif').setVolume(Tools.prefs('notifvolume')).play();
+					}
+					this.notifyOnce(row[1], "\"" + row.slice(2).join('|') + "\"", 'highlight');
```

Comment:
Alright, thanks for the clarification, and for /wall i think it shouldn't mass HL, its constantly used to grab attention in a more subtle manner. An example is the host of a mafia game (assuming they are % or up), they use /wall to announce the player list, deaths, anything of importance really. But non-players don't want to be Hled by that tbh, it would become annoying quickly.